packages/atomic/cypress/e2e/search-box/search-box-assertions.ts

```diff
@@ -78,6 +78,13 @@ export function assertSuggestionIsSelectedWithoutIt(index: number) {
     .should('contain', 'active-suggestion');
 }
 
+export function assertSuggestionIsNotSelectedWithoutIt(index: number) {
+  SearchBoxSelectors.querySuggestions()
+    .eq(index)
+    .should('have.attr', 'part')
+    .and('contain', 'active-suggestion');
```

Comment:
I'm not super used to Cypress, not having worked with it much myself, but the way this reads makes it seem like a suggestion that is not selected should contain an `active-suggestion` part, which feels pretty counter-intuitive to me 🤔 

Can you explain how this assertion works?